### DISCUSSION TITLE
[논의] '책 추가 페이지' 용어 통일에 대한 건

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#88] [General]
저희가 예전에 '책 추가 페이지'라고 부르던 페이지가 지금 프로젝트 테이블에도 '책 추가 페이지'라고 입력이 되어 있는데, 이거 누군가는 '책 등록 페이지'로 인지하고 있지 않을까 싶습니다. 저희 예전에 회의를 통해 이거 '책 등록 페이지'라고 부르기로 합의한 걸로 기억하거든요. 이 부분에 대한 용어 통일이 필요하지 않을까 싶습니다.
 그래야 시안에도 같은 용어로 페이지 이름이 표기 될 수 있을 것 같습니다. 아주 간단한 안건이라 논의하는 데에 몇 분 소요 되지 않을 것 같습니다. 편하게 의견 주시면 감사하겠습니다.

**Comment by Samuel-0930** [comment]
저는 등록, 추가 둘 다 합의만 된다면 이름은 상관 없을 것 같습니다!!

**Comment by DragonTiger92** [comment]
프로젝트 기획 회의 중 해당 사안이 논의 됨. '책 등록 페이지'로 용어를 통일하기로 결정이 되며 해당 discussion의 issue가 해결 되어 discussion을 종료함.
